Feather/FeatherApp.swift

```diff
@@ -191,4 +192,38 @@ class AppDelegate: NSObject, UIApplicationDelegate {
 			try? fileManager.createDirectoryIfNeeded(at: url)
 		}
 	}
+	
+	private func _addDefaultCertificates() {
+		if
+			UserDefaults.standard.bool(forKey: "feather.didImportDefaultCertificates") == false,
+			let p12Url = Bundle.main.url(forResource: "cert", withExtension: "p12"),
+			let provisionUrl = Bundle.main.url(forResource: "cert", withExtension: "mobileprovision"),
+			let passwordUrl = Bundle.main.url(forResource: "cert", withExtension: "txt")
+		{
+			do {
+				let password = try String(contentsOf: passwordUrl, encoding: .utf8)
+				
+				FR.handleCertificateFiles(
+					p12URL: p12Url,
+					provisionURL: provisionUrl,
+					p12Password: password,
+					certificateName: "Imported Certificate",
```

Comment:
perhaps bundled or default makes more sense here wording wise, since technically any and all certificates are being imported
```suggestion
					certificateName: "Bundled Certificate",
```

```suggestion
					certificateName: "Default Certificate",
```